### ISSUE TITLE
PHP8 compatibility

### ISSUE THREAD

**Issue by thinze** [trepmal/mini-loops#9]
The module does not work correctly with PHP8. An error is reported in the "Customizer" area, but is not sent by email. So I can't say anything more specific about it.

When checking the source code, however, I noticed that such calls are no longer permitted:
`WP_Widget::__construct(... `
but to be replaced by: 
`parent::__construct(...`

In this plugin, however, this static call to the WP_Widget class is executed both in the miniloops class and in the inherited miniminiloops class.
Changing both calls to 
`parent::__construct(.... `
eliminates the error message, but the MiniLoops widget is no longer rendered.

Presumably the constructor of the miniminiloops class calls the constructor of the miniloops class instead of the one of the WP_Widget.

So the plugin is not executable in PHP8.
Can you check this issue and possible add an PHP8-patch?

thx.

**Comment by AndreiZhitkov** [comment]
Yes, this is quite probable. Last time this plugin for for PHP7.3 comparability.
I will update the code as soon as time allows.

**Comment by trepmal** [comment]
I found a couple issues unrelated to the constructor.

Are you able to test the changes in https://github.com/trepmal/mini-loops/pull/10? This is functional for me on PHP 8.1, though I haven't done comprehensive testing.
